### ISSUE TITLE
Excessive console logs in development logs

### ISSUE THREAD

**Issue by bai** [better-auth/better-auth#619]
**Describe the bug**

When following Next.js installation guide, getting excessive warnings in dev console logs:

```
 GET /api/auth/get-session 200 in 813ms
 GET /api/auth/get-session?currentURL=http%3A%2F%2Flocalhost%3A3000%2F 200 in 514ms
 ⚠ ./node_modules/.pnpm/better-auth@1.0.0-canary.14/node_modules/better-auth/dist
Package oslo can't be external
The request oslo matches serverExternalPackages (or the default list).
The request could not be resolved by Node.js from the project directory.
Packages that should be external need to be installed in the project directory, so they can be resolved from the output files.
Try to install it into the project directory by running npm install oslo from the project directory.
```

**To Reproduce**

1. Follow [guide](https://beta.better-auth.com/docs/integrations/next)
2. Add social signin with Google.
3. Run `pnpm dev` and try logging in with google.

**Expected behavior**

Clear console logs.

**Screenshots**

**Desktop (please complete the following information):**

- OS: MacOS
- Browser: Happens server-side
- Version: 22 LTS

**Comment by avanderbergh** [comment]
This is a NextJS-specific error...

[Here](https://nextjs.org/docs/app/api-reference/next-config-js/serverExternalPackages) we can see that the `olso` package is included in the default list of packages not bundled by Next.

So, you should probably add `olso` as a direct dependency to your project?

**Comment by bai** [comment]
Yep I'm doing that! Decided to report in case that was something to look into (code-wise or docs-wise).
